sycl/include/CL/sycl/types.hpp

```diff
@@ -50,17 +50,12 @@
 #include <CL/sycl/multi_ptr.hpp>
 
 #include <array>
+#include <cfenv>
```

Comment:
Please, put this include under `#ifndef __SYCL_DEVICE_ONLY__`.

Reply:
done.